mailpoet/assets/js/src/automation/editor/store/register_step_type.ts

```diff
@@ -1,7 +1,10 @@
 import { dispatch } from '@wordpress/data';
+import { Hooks } from 'wp-js-hooks';
 import { storeName } from './constants';
 import { StepType } from './types';
 
 export const registerStepType = (stepType: StepType): void => {
-  dispatch(storeName).registerStepType(stepType);
+  dispatch(storeName).registerStepType(
+    Hooks.applyFilters('mailpoet.register_step_type', stepType),
```

Comment:
We could use such a hook to extend action stubs.

Reply:
Should we add "automation", such as `mailpoet.automation.register_step_type`?